README.md

```diff
@@ -1,3 +1,5 @@
+[![Gitpod ready-to-code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/sourcegraph/src-cli)
```

Comment:
This should be inline with the other badges below, not on a separate line at the top as it is here